modules/mbedtls/register_types.cpp

```diff
@@ -35,6 +35,10 @@
 #include "packet_peer_mbed_dtls.h"
 #include "stream_peer_mbedtls.h"
 
+#ifdef TESTS_ENABLED
```

Comment:
This does nothing in `3.x`, and the referenced header file is not present anyway. should be removed.

Reply:
Thanks!